app_test.go

```diff
@@ -221,6 +221,60 @@ func ExampleApp_Run_subcommandNoAction() {
 
 }
 
+func ExampleApp_Run_bashComplete_withShortFlag() {
+	os.Args = []string{"greet", "-", "--generate-bash-completion"}
+
+	app := NewApp()
+	app.Name = "greet"
+	app.EnableBashCompletion = true
+	app.Flags = []Flag{
+		IntFlag{
+			Name: "other,o",
+		},
+		StringFlag{
+			Name: "xyz,x",
+		},
+	}
+
+	app.Run(os.Args)
+	// Output:
+	// --other
+	// -o
+	// --xyz
+	// -x
+	// --help
+	// -h
+	// --version
+	// -v
+}
```

Comment:
These tests do not assert anything. Why can't we set a custom writer and assert the output?

Reply:
// Output:
is test right? it is expecting output to match

Reply:
First time I am hearing that. Is that a go thing or our magical test runner? (I am new to this library)

Reply:
:astonished: been writing go since 2013, did not know this.